test/integration_test/lib/util/test_author.py

```diff
@@ -0,0 +1,11 @@
+from lib.util.author import *
+
+def test_get_uid_map():
+
+	author_uid_map=dict()
+	check='jdelvare@suse.de' in author_uid_map
+	assert check==False
+	
+	author_uid_map=get_uid_map()
```

Comment:
The module takes `clean_data.json` file as input and writes output to `author_uid_map.json`. There is also a binary argument named `write_to_file` which remains unused. You can remove the parameter from the function definition on [line-6](https://github.com/DeveloperCAP/MLCAT/blob/master/lib/util/author.py#L6). 

The test must provide a really small `clean_data.json` file as input and check the returned value against expected dictionary of author_uid_map. You should also match the saved file against another reference file to see if the function is writing to the output file ( `author_uid_map.json`) correctly.

The necessary data files for the test can be stored in the [integration/data](https://github.com/DeveloperCAP/MLCAT/tree/master/test/integration_test/data) directory.

Reply:
There is already a `clean_data.json` file in development branch https://github.com/DeveloperCAP/MLCAT/blob/development/test/integration_test/data/clean_data.json

You can take the json file paths as parameters which would also ease the tasks of data separation. Just check for the dependencies of `author.py` and change them accordingly. The new function can look like `get_uid_map(input_file, output_file)`